epoxy-annotations/src/main/java/com/airbnb/epoxy/OnVisibilityChanged.java

```diff
@@ -0,0 +1,18 @@
+package com.airbnb.epoxy;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * This can be used to annotate methods inside classes with a {@link ModelView} annotation. Methods
+ * with this annotation will be called when the viewport is
```

Comment:
I'm not clear on what changes to the viewport mean in this context. can this documentation link to the epoxy model method that this maps to?

would be nice to update the link to point to epoxy specific documentation when we have it